### ISSUE TITLE
[V3] [3.0.2] Vimeo + Autoplay doesn't hide the controls in IE + Firefox

### ISSUE THREAD

**Issue by MBWebTechMariusz** [sampotts/plyr#825]
### Expected behaviour
When Vimeo is used and autoplay is on, the controls should hide as soon as the video starts.

### Actual behaviour
The controls are visible, and the video has started

### Environment

- Browser: IE + Firefox
- Version: Latest
- Operating System: Windows
- Version: 8.1

### Steps to reproduce
See behaviour; works in Chrome though.
I will take a deeper look if there some different events.

**Comment by MBWebTechMariusz** [comment]
Couldn't fix the issue in the code. Needed to change the autoplay mechanism:

```
        var player = new Plyr(videoPlayer, {
            autoplay: false
        });

        // Fix for autoplay
        player.on('ready', event => {
            if (videoModel["autoplay"]) {
                player.play();
            }
        });
```

**Comment by MBWebTechMariusz** [comment]
Maybe this has something to do with this pull request: https://github.com/sampotts/plyr/pull/805/files

**Comment by sampotts** [comment]
That’s old v2 code so doubtful.

**Comment by sampotts** [comment]
Can you try again? I think this may be fixed in `v3.0.11` as I can't reproduce it now.
